app/_src/quickstart/docker-demo.md

```diff
@@ -0,0 +1,716 @@
+      ```
+
+4. **Check if service is running**
+
+   To confirm the service is set up correctly and running, use the {{ kumactl }} to inspect the services:
+
+   ```sh
+   kumactl inspect services
+   ```
+
+   The output should show a single service, `kv`, with the status `Online`.
+
+   You can also open the [{{ Kuma }} GUI]({{ docs }}/production/gui/) at <http://127.0.0.1:25681/gui/meshes/default/services/mesh-services>. Look for the `kv` service, and verify that its state is `Available`.
+
+### Demo Application
+
+The steps are the same as those explained earlier, with only the names changed. We won’t repeat the explanations here, but you can refer to the [Key/Value Store service](#keyvalue-store) instructions if needed.
+
+1. **Generate a data plane token**
+
+   ```sh
+   kumactl generate dataplane-token \
+     --tag kuma.io/service=demo-app \
+     --valid-for 720h \
+     > {{ tmp }}/token-demo-app
+   ```
+
+2. **Start the application container**
+
+   ```sh
+   docker run \
+     --detach \
+     --name {{ kuma-demo }}-app \
+     --hostname demo-app \
+     --network {{ kuma-demo }} \
+     --ip {{ ip_abc }}.3 \
+     --publish 25050:5050 \
+     --volume {{ tmp }}:/demo \
+     --env KV_URL=http://kv.svc.mesh.local:5050 \
+     --env APP_VERSION=v1 \
+      ghcr.io/kumahq/kuma-counter-demo:debian-slim
+   ```
+
+3. **Prepare the application container**
+
+   Enter the container to run further commands.
+
+   {% if version == "preview" %}
+   ```sh
+   docker exec \
+     --tty \
+     --interactive \
+     --privileged \
+     --env {{ KUMA_PREVIEW_VERSION }} \
+     {{ kuma-demo }}-app bash
+   ```
+   {% else %}
+   ```sh
+   docker exec --tty --interactive --privileged {{ kuma-demo }}-app bash
+   ```
+   {% endif %}
+
+   {{ warning-run-inside-container | indent }}
+
+   1. **Install tools and create data plane proxy user**
+
+      {{ code-block-install-tools-create-user | indent | indent }}
+
+   2. **Start the data plane proxy**
+
+      ```sh
+      runuser --user {{ kuma-data-plane-proxy }} -- \
+        /usr/local/bin/kuma-dp run \
+        --cp-address https://control-plane:5678 \
+        --dataplane-token-file /demo/token-demo-app \
+        --dataplane-file /demo/dataplane.yaml \
+        --dataplane-var name=demo-app \
+        --dataplane-var address={{ ip_abc }}.3 \
+        --dataplane-var port=5050 \
+        > /demo/logs-data-plane-proxy-demo-app.log 2>&1 &     
+      ```
+
+   3. **Install the transparent proxy**
+
+      {{ warning-tproxy-command-run-inside-container | indent | indent }}
+
+      ```sh
+      kumactl install transparent-proxy \
+        --config-file /demo/config-transparent-proxy.yaml \
+        > /demo/logs-transparent-proxy-install-demo-app.log 2>&1
+      ```
+
+   4. **Exit the container**
+
+      Demo application is now set up and running. You can safely exit the container as the configuration is complete:
+
+      ```sh
+      exit
+      ```
+
+4. **Verify the application**
```

Comment:
should we also add step witch checking services via kumactl to show that it was registered in mesh?

Reply:
addressed